test/test_helper.rb

```diff
@@ -13,11 +14,26 @@
 ENV['KEYCLOAK_SITE_URL'] = 'http://www.example.com/auth/keycloak/callback'
 require_relative '../config/environment'
 require 'rails/test_help'
+require 'vcr'
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'test/vcr_cassettes'
+  config.hook_into :webmock
+  config.allow_http_connections_when_no_cassette = true
+  config.filter_sensitive_data('github_token') { ENV['GITHUB_ACCESS_TOKEN'] }
+  config.filter_sensitive_data('github_client_id') { ENV['GITHUB_CLIENT_ID'] }
+  config.filter_sensitive_data('github_client_secret') { ENV['GITHUB_CLIENT_SECRET'] }
+  # whitelist 127.0.0.1 so VCR doesn't interfere with system tests
+  config.ignore_hosts '127.0.0.1'
+end
 
 module ActiveSupport
   class TestCase
     # Run tests in parallel with specified workers
-    parallelize(workers: :number_of_processors)
+
+    # Don't parallelize because there is an issue with simplecov
+    # See: https://github.com/simplecov-ruby/simplecov/issues/718
+    # parallelize(workers: :number_of_processors)
```

Comment:
I thought the code coverage was working? 🤔

Reply:
Sometimes is works and sometimes it doesn't :man-shrugging: We couldn't figure out why but I'm thinking it has something to do with the number of cores on the github action process. 

For some reason, coverage works for me locally with parallelization but it doesn't for Riley.  I'm pretty sure we have a different number of cores on our machines

Reply:
Ohhh. Yeah I remember something about parallelize won't kick in until you have at least X number of tests (50?).